File: src/main/java/com/redgrapefruit/mythicaltowers/common/item/IngotItem.java
package com.redgrapefruit.mythicaltowers.common.item;

import com.redgrapefruit.mythicaltowers.common.init.MythicalItemGroups;
import net.minecraft.item.Item;

/**
 * A ingot of a certain custom material
 */
public class IngotItem extends Item {
    public IngotItem() {
        super(new Item.Settings().group(MythicalItemGroups.INGOTS));
    }
}

File: src/main/kotlin/com/redgrapefruit/mythicaltowers/common/item/IngotItem.kt
package com.redgrapefruit.mythicaltowers.common.item

import com.redgrapefruit.mythicaltowers.common.registry.MythicalItemGroups
import net.minecraft.item.Item

/**
 * A ingot of a certain custom material
 */
class IngotItem : Item(Settings().group(MythicalItemGroups.INGOTS))